c/cert/test/rules/ERR32-C/test.c

```diff
@@ -0,0 +1,42 @@
+#include <errno.h>
```

Comment:
do we want to add the other COMPLIANT case, as like a self documenting type thing?
```
void handlerX(int signum) {
  pfv old_handler = signal(signum, SIG_DFL);
  if (old_handler == SIG_ERR) {
    _Exit(0); // COMPLIANT
  }
}
```